sportsreference/nba/roster.py

```diff
@@ -1206,7 +1206,8 @@ def __init__(self, team, year=None, slim=False):
         else:
             self._players = []
 
-        self._find_players(year)
+        self.coach = None
```

Comment:
Same as with the MLB comment above, but also adding for this one that I think this would be best right after line 1205 between the `self._slim = slim` and `if slim:` lines.